ai-ml/jark/terraform/destroy.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
Let's rename the file from `destroy.sh` to  `cleanup.sh` for consistency across all the blueprints